packer/nomad-server-client/image.pkr.hcl

```diff
@@ -80,6 +56,40 @@ variable "aws_build_region" {
   default     = "us-east-1"
 }
 
+####################
+# Program Versions
```

Comment:
i couldn't figure out what to call this. Dependency versions?

Reply:
"Software Versions"